apps/estuary/lib/estuary/message_handler.ex

```diff
@@ -1,8 +1,33 @@
 defmodule Estuary.MessageHandler do
+  @moduledoc """
+  This module will read an event from the event stream and persist it.
```

Comment:
```suggestion
  MessageHandler reads an event from the event stream and persists it.
```

Reply:
We should document what something does today as opposed to what it may/will do in the future.

Reply:
Agreed good catch